### ISSUE TITLE
FP16 training error

### ISSUE THREAD

**Issue by Retiina** [TuSimple/simpledet#120]
I want to train `mask_r50v1_fpn` in FP16 mode, so I modify `config.general.fp16 = True`. But when I begin to train, there is an error:

```
Traceback (most recent call last):
  File "detection_train.py", line 272, in <module>
    train_net(parse_args())
  File "detection_train.py", line 22, in train_net
    transform, data_name, label_name, metric_list = config.get_config(is_train=True)

  File "/simpledet/simpledet/config/benchmark/mask_r50v1_fpn_1x_fp16.py", line 171, in get_config mask_head)

  File "/simpledet/simpledet/models/maskrcnn/builder.py", line 37, in get_train_symbol bbox_loss = bbox_head.get_loss(roi_feat, bbox_cls, bbox_target, bbox_weight)

  File "/simpledet/simpledet/symbol/builder.py", line 387, in get_loss
    cls_logit, bbox_delta = self.get_output(conv_feat)

  File "/simpledet/simpledet/symbol/builder.py", line 326, in get_output
    head_feat = X.to_fp32(head_feat, name="bbox_head_to_fp32")

  File "/simpledet/simpledet/mxnext/simple.py", line 386, in <lambda>
    to_fp32 = lambda data, name: mx.sym.cast(data, dtype=np.float32, name=name)

  File "<string>", line 46, in cast
AssertionError: Argument data must be Symbol instances, but got {'classification': <Symbol bbox_fc2_relu>, 'regression': <Symbol bbox_fc2_relu>}
```

Could you give me some advance? Thanks in advance!

**Comment by xchani** [comment]
Seems to be a bug caused by pr #116 . @RogerChern

**Comment by xchani** [comment]
Fixed in #122
